components/CanarySearch.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+export default function CanarySearch() {
+  const [loaded, setLoaded] = React.useState(false);
+
+  const TABS = JSON.stringify([
+    {
+      name: "Docs",
+      pattern: "**/langfuse.com/**",
+    },
+    {
+      name: "GitHub",
+      pattern: "**/github.com/**",
+    },
+  ]);
+
+  React.useEffect(() => {
```

Comment:
is this necessary or could we use next dynamic imports to achieve the same result?

https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading#nextdynamic

Reply:
I think async loading is good here, 90kb is kinda small but needs to be async/lazy in some form and not part of the core bundle.

Reply:
https://github.com/user-attachments/assets/58700dcb-0d41-411e-b8a6-a71865c6e83a

Did some research on this, and it seems it will be not part of the core bundle.
Since `import()` is placed only inside `useEffect`, `webpack` make separate bundle for each component, and load them after other part of the app(in this case, `FeatureBento.tsx`, `Pricing.tsx`, etc...)